discovery-provider/src/utils/ipfs_lib.py

```diff
@@ -40,32 +40,33 @@ def get_metadata(self, multihash, metadata_format, user_replica_set=None):
         """
         logger.warning(f"IPFSCLIENT | get_metadata - {multihash}")
         api_metadata = metadata_format
-        retrieved_from_local_node = False
-        retrieved_from_gateway = False
+        retrieved = False
```

Comment:
nit - let's leave this as `retrieved_metadata`

Reply:
i'll use this as `retrieved_from_SOURCE` since we're keeping in the logs for which source(s) succeeded/failed